tutorials/audio/text_to_speech.rst

```diff
@@ -0,0 +1,71 @@
+.. _doc_text_to_speech:
+
+Text to speech
+==============
+
+Basic Usage
+-----------
+
+Basic usage of text-to-speech involves the following one-time steps:
+
+- Query the system for a list of usable voices
+- Store the ID of the voice you want to use
+
+Once you have the voice ID, you can use it to speak some text:
+
+::
+
+    # One-time steps
+    var voices:Array = DisplayServer.tts_get_voices()
+    # Pick a voice. Here, we arbitrarily pick the first voice.
+    var voice_data:Dictionary = voices[0]
```

Comment:
While this is true on Linux, when I tried this on Windows, I got two voices, and I don't recall that either had an indication of the language (English?).

On Linux, I got ~13k languages.

Reply:
All voices should have "language", "name", and "id" (it might be all English, if nothing else is installed). If "language" is empty, there might be a bug. It's definitely returning correct languages (`xx_XX` codes) on my Windows pc.